src/outputs/outputs.cpp

```diff
@@ -465,228 +375,4 @@ void Outputs::MakeOutputs(Mesh *pm, ParameterInput *pin, SimTime *tm,
   Kokkos::Profiling::popRegion(); // MakeOutputs
 }
 
-//----------------------------------------------------------------------------------------
-//! \fn void OutputType::TransformOutputData(MeshBlock *pmb)
-//  \brief Calls sum and slice functions on each direction in turn, in order to allow
-//  mulitple operations performed on the same data set
-
-bool OutputType::TransformOutputData(MeshBlock *pmb) {
```

Comment:
This is machinery we may want to re-enable at some point, but I'm not convinced here is the place and this is the way. Some food for thought:
- In an AMR hierarchy, is slicing or summing a meaningful operation?
- What about interpolating to a reduced-dimensionality separate grid?
- Should this be machinery the downstream code can register in some way to specify, e.g., specific integration weights?
- Should we be able to do this for all formats? Only HDF5? Only ASCII?

Reply:
> This is machinery we may want to re-enable at some point, but I'm not convinced here is the place and this is the way. Some food for thought:

It may be worth to revive that discussion after people got the chance to experiment with Ascent for a bit.
As far as I can tell all that machinery will be automatically in place (though I don't know how "fast" it'll be and also how where the emphasis should be between duplicated, but internal code versus an external potentially not easy to install external dependency).
 
>     * In an AMR hierarchy, is slicing or summing a meaningful operation?

Definitely. (I know we'll need it)

>     * What about interpolating to a reduced-dimensionality separate grid?

Likely useful, but not sure it's actually required internally as the plotting/postprocessing (like interpolation) can easily be handled by external tools
 
>     * Should this be machinery the downstream code can register in some way to specify, e.g., specific integration weights?

Might also be useful. Another solution could be to allow for full function callsbacks (similar to the hst outputs).

>     * Should we be able to do this for all formats? Only HDF5? Only ASCII?

I'm happy with plain hdf5 and providing easy python script to read/process/plot/convert the data based on the assumption that this will cover the use cases of a big majority of the users.

Reply:
> It may be worth to revive that discussion after people got the chance to experiment with Ascent for a bit.
As far as I can tell all that machinery will be automatically in place (though I don't know how "fast" it'll be and also how where the emphasis should be between duplicated, but internal code versus an external potentially not easy to install external dependency).

Oh interesting. I think that probably won't work for Phoebus, since it does things with metrics. Yeah I don't think we need to tackle this now. I just had it at the mind as I was culling this code. :)

>  I'm happy with plain hdf5 and providing easy python script to read/process/plot/convert the data based on the assumption that this will cover the use cases of a big majority of the users.

I think we may want to provide in-situ reduced dimensionality output---it can be very convenient and save time- and memory-consuming steps. But I also don't know if I've entirely thought through what the right thing here is.

> > In an AMR hierarchy, is slicing or summing a meaningful operation?

> definitely. (I know we'll need it)

Can you elaborate? I can see how slice/sum operations over the whole grid are meaningful, but slicing/summing on a per-meshblock basis doesn't seem to provided what we need unless one assumes, e.g., symmetry in the sliced direction. Maybe I'm missing something.

Reply:
it buys me easy, high cadence, cheap outputs for movies :)